src/Protector.php

```diff
@@ -363,20 +377,28 @@ protected function generateDump(string $destinationFilePath, array $options = []
 
         $this->createDirectory(Storage::disk('local')->path(dirname($destinationFilePath)));
 
-        try {
-            // Write dump using specific options.
-            exec(sprintf('mysqldump %s > %s 2> /dev/null',
+        $this->isExecEnabled();
```

Comment:
Let's execute code and create a directory, even when we cant do anything with it.

+1

Reply:
Fixed.